CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Default owners for everything in this repo
+* @adfinis-sygroup
```

Comment:
IMO this group is way to big, please consider creating a dedicated team for use in CODEOWNERS.

Reply:
thanks, I changed it to a smaller team